phpunit/bootstrap.php

```diff
@@ -64,10 +64,16 @@
 /**
  * Manually load the plugin being tested.
  */
-function _manually_load_plugin() {
+function _manually_load_plugins() {
 	require dirname( __DIR__ ) . '/lib/load.php';
+
+	$plugin_dirs = glob( dirname( __DIR__ ) . '/plugins/*', GLOB_ONLYDIR );
+	foreach ( $plugin_dirs as $dir ) {
+		$plugin_name = basename( $dir );
+		require "$dir/$plugin_name.php";
+	}
 }
-tests_add_filter( 'muplugins_loaded', '_manually_load_plugin' );
+tests_add_filter( 'muplugins_loaded', '_manually_load_plugins' );
```

Comment:
Do we want to run all the tests of Gutenberg in the same CI job or we want to separate the CI jobs for plugins?

Reply:
Same for JS and e2e tests.

Reply:
Good question: it would probably be better to have separate CI jobs, to help ensure all plugins work independently from Gutenberg (and from one another). Though I think all jobs should be run on all PRs, to catch any Gutenberg changes that accidentally break unrelated things in plugins.